### ISSUE TITLE
A better 404?

### ISSUE THREAD

**Issue by bpierre** [lesintegristes/lesintegristes-theme#13]
http://www.lesintegristes.net/404

**Comment by gregoiredierendonck** [comment]
Perhaps can we have a good stuff here...

Don't know, using SVG or Canvas for showing in a graphic how many people have visited this page (dynamic)...
Or using a map with a message "Are you lost?", and using, in the maps graphic, the localisation of the user...

Other ideas?

**Comment by bpierre** [comment]
> Don't know, using SVG or Canvas for showing in a graphic how many people have visited this page (dynamic)...

Could work well with the real time header.

**Comment by kaneel** [comment]
Sounds like a d3.js task!

**Comment by bluety** [comment]
Hello, 
Est-on obligé de parler anglais :( ?
Pourquoi pas soutenir ce projet : http://notfound.org/
Ça affiche une photo d'un enfant disparu.

**Comment by kaneel** [comment]
We could as well use the 404 pages to display our resumes :p

ps: we speak english because inside us, since 911, we're all americans.

Le 29 septembre 2012 13:06, Bluety notifications@github.com a écrit :

> Hello,
> Est-on obligé de parler anglais :( ?
> Pourquoi pas soutenir ce projet : http://notfound.org/
> Ça affiche une photo d'un enfant disparu.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/lesintegristes/lesintegristes-theme/issues/13#issuecomment-9002975.

## 

// Guillaume *kaneel *Richard http://mynameiskaneel.com
// With many lower than 3

**Comment by ericlebihan** [comment]
Personnellement je préfère que les intégristes restent indépendants et
associés à aucune organisation soit-elle à but humanitaire ou caritatif. Ce
qui n'empêche pas un engagement personnel de ceux qui le souhaitent.

2012/9/29 Bluety notifications@github.com

**Comment by kaneel** [comment]
We also can post a random track of mine, taken from
https://soundcloud.com/kaneel :)

Le 29 septembre 2012 13:17, ericlebihan notifications@github.com a écrit :

> Personnellement je préfère que les intégristes restent indépendants et
> associés à aucune organisation soit-elle à but humanitaire ou caritatif.
> Ce
> qui n'empêche pas un engagement personnel de ceux qui le souhaitent.
> 
> 2012/9/29 Bluety notifications@github.com
> 
> > Hello,
> > Est-on obligé de parler anglais :( ?
> > Pourquoi pas soutenir ce projet : http://notfound.org/
> > Ça affiche une photo d'un enfant disparu.
> > 
> > —
> > Reply to this email directly or view it on GitHub<
> > https://github.com/lesintegristes/lesintegristes-theme/issues/13#issuecomment-9002975>.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/lesintegristes/lesintegristes-theme/issues/13#issuecomment-9003025.

## 

// Guillaume *kaneel *Richard http://mynameiskaneel.com
// With many lower than 3

**Comment by ericlebihan** [comment]
Personally I prefer that les integristes remain independent and not
associated with any organization as it is for humanitarian or charitable.
This does not preclude a personal commitment of those who wish.

2012/9/29 Eric Le Bihan eric.ericlebihan@gmail.com

**Comment by bpierre** [comment]
@bluety

Non, pas obligé de parler anglais, mais nous essayons de le faire au maximum pour être ouverts aux contributeurs non francophones.

Cette initiative est intéressante, mais l’idée part du principe que les pages 404 servent peu ou pas. Je pense au contraire que même lorsque la page est principalement une « blague » (exemple : https://github.com/404), cela fait partie de l’image du site, et c’est encore plus important pour un site qui s’adresse à un public d’utilisateurs « avancés ». Ça participe à poser l’identité du site, au même titre que le reste.

Ils devraient proposer cette intégration pour ceux qui le souhaitent sans orienter ça « 404 » (mais toute leur identité est basée là-dessus, et le service attirerait moins l’attention…). On devrait pouvoir l’intégrer sous forme de bannière par exemple. Parce qu’il y a un truc qui ne sert **vraiment** à rien sur beaucoup de sites : les bannières publicitaires. :-)

Et la vraie raison, comme déjà indiqué par les autres : pas de réclame sur Les integristes, peu importe le but :-)
